src/private/bot.ts

```diff
@@ -66,11 +66,21 @@ export namespace bot {
   }
 
   export interface QueryResponse {
+    data: BotResults | BotAuth;
+    type: BotResponseType;
+  }
+
+  export interface BotResults {
```

Comment:
we can remove Bot prefix from all interface as all these interfaces are already inside bot namespace